src/DashboardHub/Bundle/AppBundle/Menu/MenuBuilder.php

```diff
@@ -55,6 +55,7 @@ public function createMainMenu(Request $request)
             );
         } else {
             $menu->addChild('Login', array('route' => 'login'));
+            $menu->addChild('Mock Login', array('route' => 'dashboard_hub_development_login'));
```

Comment:
Hmm not sure where we will put this? Might not actually need it, as we (or Behat) can go directly to the url when in `dev/test` mode.